src/Analyzers/KnownTypeSymbols.cs

```diff
@@ -50,6 +55,31 @@ public sealed class KnownTypeSymbols(Compilation compilation)
     /// </summary>
     public INamedTypeSymbol? DurableTaskRegistry => this.GetOrResolveFullyQualifiedType("Microsoft.DurableTask.DurableTaskRegistry", ref this.durableTaskRegistry);
 
+    /// <summary>
+    /// Gets a TaskOrchestrationContext type symbol.
+    /// </summary>
+    public INamedTypeSymbol? TaskOrchestrationContext => this.GetOrResolveFullyQualifiedType("Microsoft.DurableTask.TaskOrchestrationContext", ref this.taskOrchestrationContext);
+
+    /// <summary>
+    /// Gets a DurableClientAttribute type symbol.
+    /// </summary>
+    public INamedTypeSymbol? DurableClientAttribute => this.GetOrResolveFullyQualifiedType("Microsoft.Azure.Functions.Worker.DurableClientAttribute", ref this.durableClientAttribute);
```

Comment:
Might be worth splitting functions specific types into a sub class `KnownTypeSymbols.Functions.{Symbol}`. Can also then organize into partial-files.

Are there other groupings of known types we can split into sub classes?

Reply:
Great suggestion, thanks!
For now, I did a quick refactor into partial classes. In another PR, I am going to refactor it to group into subclasses, I loved the suggestion.